### DISCUSSION TITLE
Unable to authenticate with password

### DISCUSSION THREAD

**Discussion by BeanHeaDza** [pocketbase/pocketbase#1259] [Q&A] [answered]
Hi guys, I'm new to pocketbase and I'm sure I'm doing something wrong here. I'm trying to login with a user I just created, but I am unable to auth with a password. I am using the JS SDK on NodeJS.

``` typescript
const pb = new PocketBase("http://127.0.0.1:8090");
await pb.collection("users").create({
    username: "test",
    password: "asdqwe123",
    passwordConfirm: "asdqwe123",
});

// The auth command gives the following error:
// "ClientResponseError 400: Failed to authenticate."
const auth = await pb.collection("users").authWithPassword("test", "asdqwe123");
return auth;
```

**Comment by ganigeorgiev** [comment, marked as answer]
Have you enabled "Username/Password authentication" for the users collection (it is under the collection options tab)?

**Comment by BeanHeaDza** [comment]
I feel silly now. I forgot I turned it off, because I wanted to do auth2, but the provider I'm using isn't listed.
